mock/environment/__init__.py

```diff
@@ -8,8 +12,36 @@
 
 from common.minimodel import Site, Variant, CloudCover, ImageQuality, WaterVapor
 
+logging.basicConfig(level=logging.INFO)
```

Comment:
This line should only be in the code where we have:
```
if __name__ == '__main__':
    # Code goes here
```
Don't worry about it now. If you set it up here, though, it will affect the whole Scheduler, which we don't want. We only want to set the logging level if the Env historical data is running separately.